### PR TITLE
Update zero react frontend lx

### PR DESCRIPTION
**Pull request by dtoki**
Updates the readmes for the `root/` and `template/` of the zero react frontend application.

### CODE REVIEW

File: templates/README.md [hunk 1]
```diff
@@ -1,91 +1,32 @@
...
-## Backend App
+## Circle CI
 
-To run the backend app locally all you need to do is build the container and run it alongside your app.
+Your repository comes with a end-to-end CI/CD pipeline, which includes the following steps:
 
-### Backend Repo
+1. Checkout
+2. Unit Tests
+3. Build for staging & production
```
Comment (davidcheung) [inline templates/README.md:21]: the frontend actually has 2 separate builds for stage / prod
might be worth it to say the deploy steps includes 
- aws s3 sync
- cloudfront invalidation

File: README.md [hunk 2]
```diff
@@ -1,12 +1,72 @@
...
+
+___
+
+## Initialization
+
+This step is meant to be executed during `zero apply`, includes following steps:
+
+- Adding environment variables to CircleCI project.
+- Linking the CircleCi with the GitHub repository
+  - Linking the circleCI will automatically trigger the first build and deploy your application to AWS S3
```
Comment (davidcheung) [inline README.md:57]: should probably talk about cloudfront and explain the wait time can be long

File: README.md [hunk 3]
```diff
@@ -1,12 +1,72 @@
...
+
+```sh
+/   # file in the root directory is for initializing the user's repo and declaring metadata
+|-- Makefile                        #make command triggers the initialization of repository
+|-- zero-module.yml                 #module declares required parameters and 
+|   # files in templates become the repo for users
+|   templates/
+|   |
+|   |-- docs/
+|   |   | create-react-app.md       #contains documentation on how work with your react application and external links to developer resources.
```
Comment (davidcheung) [inline README.md:28]:
```suggestion
|   |   | create-react-app.md       #documentation on how to work with your react application and external developer resources.
```

File: README.md [hunk 4]
```diff
@@ -1,12 +1,72 @@
...
 
-This is a [zero](https://github.com/commitdev/zero) module which sets up a
-service which can be deployed to the environment set up with [zero-aws-eks-stack](https://github.com/commitdev/zero-aws-eks-stack).
-It also contains a simple CircleCI pipeline which defines how to build and deploy the assets.
+This is a [Zero] module which sets up a
+service which can be deployed to the environment set up with [zero-aws-eks-stack][zero-infra].
+
+The `/templates` folder is meant to be filled in via [Zero][zero] and results in Simple React application. It also contains a simple CircleCI pipeline which defines how to build and deploy the service.
+
+This repository is framework/business-logic agnostic; mainly showcasing some universal best practices:
```
Comment (davidcheung) [inline README.md:8]:
```suggestion
This repository is business-logic agnostic; mainly showcasing some universal best practices:
```

File: README.md [hunk 5]
```diff
@@ -1,12 +1,72 @@
...
+
+- Creating an optimized build of your frontend application.
+
+- Out of the box CI/CD flow CircleCi.
+  - testing
+  - uploading build to an AWS S3 bucket.
```
Comment (davidcheung) [inline README.md:14]: - building your web-app as a static distribution in /dist folder 
- syncs the built bundle to AWS s3
- invalidates Cloudfront cached version

File: README.md [hunk 6]
```diff
@@ -1,12 +1,72 @@
-## Project Board
-https://app.zenhub.com/workspaces/commit-zero-5da8decc7046a60001c6db44/board?filterLogic=any&repos=203630543,247773730,257676371,258369081
-
 # Zero Deployable Frontend
 
-This is a [zero](https://github.com/commitdev/zero) module which sets up a
-service which can be deployed to the environment set up with [zero-aws-eks-stack](https://github.com/commitdev/zero-aws-eks-stack).
-It also contains a simple CircleCI pipeline which defines how to build and deploy the assets.
+This is a [Zero] module which sets up a
+service which can be deployed to the environment set up with [zero-aws-eks-stack][zero-infra].
```
[the remark this answers is not in the document]
Comment (dtoki) [reply README.md:4]: not sure about this suggestion, do you asking for the description to start from `line 4`?

[the remark this answers is not in the document]
Comment (davidcheung) [reply README.md:4]: sorry I wrote something else must have messed it up somehow
```suggestion
React webapp bundled statically served through CloudFront and S3 from the [zero-aws-eks-stack][zero-infra].
```

**Pull request merged**